common/helpers/field.js

```diff
@@ -19,6 +20,25 @@ function mapReferenceDataToOption({ id, title, key, conditional }) {
   return option
 }
 
+function mapAssessmentQuestionToTranslation(item) {
+  const { category, key } = item
```

Comment:
Was about to say "What about destructuring the arguments instead?" but then realised you need to use `item` later on...